sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -1860,25 +1870,6 @@ Page<NotificationHook> NextPageFunc(string nextLink, int? pageSizeHint)
             return PageableHelpers.CreateEnumerable(FirstPageFunc, NextPageFunc);
         }
 
-        private static void ValidateHookToCreate(NotificationHook hook, string paramName)
-        {
-            Argument.AssertNotNull(hook, paramName);
-            Argument.AssertNotNullOrEmpty(hook.Name, $"{paramName}.{nameof(hook.Name)}");
-
-            if (hook is EmailNotificationHook emailHook)
-            {
-                Argument.AssertNotNullOrEmpty(emailHook.EmailsToAlert, $"{paramName}.{nameof(EmailNotificationHook.EmailsToAlert)}");
-            }
-            else if (hook is WebNotificationHook webHook)
-            {
-                Argument.AssertNotNull(webHook.Endpoint, $"{paramName}.{nameof(WebNotificationHook.Endpoint)}");
```

Comment:
The `Endpoint` is required in the constructor now, so there's no much value in validating it here. A user could do:
```cs
var hook = new WebHook("<name>", new Uri(...)); // There is a null check for the URI in the ctor
hook.Endpoint = null;

adminClient.CreateHook(hook);
```
and we wouldn't catch this error. I thought a bit about this one, but I think it's not worth trying to be bullet-proof. The required parameter being in the constructor seems enough to guide users to the correct setup. Thoughts?